content/docs/reference/builders-reference.md

```diff
@@ -12,14 +12,15 @@ Paketo Builders How To [documentation][howto/builders].
-| Buildpackless Jammy Tiny Builder | Ubuntu 22.04, scratch         | No buildpacks included. Compatible with the same buildpacks as the Jammy Tiny Builder. | `paketobuildpacks/builder-jammy-buildpackless-tiny`         | https://github.com/paketo-buildpacks/builder-jammy-buildpackless-tiny         |
+| Name                             | Stack                | Use cases                                                                                            | DockerHub Distribution                        | Github Repository                                              |
+|----------------------------------|----------------------|------------------------------------------------------------------------------------------------------|-----------------------------------------------|----------------------------------------------------------------|
+| Jammy Full Builder               | Ubuntu 22.04         | PHP, Java, Node.js, Go, Python, .NET Core, Ruby, NGINX, and HTTPD buildpacks and common C libraries. | `paketobuildpacks/builder-jammy-full`         | https://github.com/paketo-buildpacks/builder-jammy-full        |
+| Buildpackless Jammy Full Builder | Ubuntu 22.04         | No buildpacks included. Compatible with the same buildpacks as the Jammy Full Builder.               | `paketobuildpacks/builder-jammy-buildpackless-full`         | https://github.com/paketo-buildpacks/builder-jammy-buildpackless-full |
+| Jammy Base Builder               | Ubuntu 22.04         | Java, Node.js, Go, Python, .NET Core, Ruby, and NGINX buildpacks without common C libraries.         | `paketobuildpacks/builder-jammy-base`         |  https://github.com/paketo-buildpacks/builder-jammy-base                                        |
+| Buildpackless Jammy Base Builder | Ubuntu 22.04         | No buildpacks included. Compatible with the same buildpacks as the Jammy Base Builder.               | `paketobuildpacks/builder-jammy-buildpackless-base`         | https://github.com/paketo-buildpacks/builder-jammy-buildpackless-base |
+| Jammy Tiny Builder               | Ubuntu 22.04, scratch | Consists of buildpacks to build most Go and Java GraalVM Native Image apps.                          | `paketobuildpacks/builder-jammy-tiny`         |  https://github.com/paketo-buildpacks/builder-jammy-tiny                                        |
+| Buildpackless Jammy Tiny Builder | Ubuntu 22.04, scratch | No buildpacks included. Compatible with the same buildpacks as the Jammy Tiny Builder.               | `paketobuildpacks/builder-jammy-buildpackless-tiny`         | https://github.com/paketo-buildpacks/builder-jammy-buildpackless-tiny |
+| UBI8 Builder                     | RHEL 8               | Java, Node.js buildpacks and extensions to install rpms.                                             | `paketobuildpacks/builder-ubi8-base`         | https://github.com/paketo-buildpacks/builder-ubi8-base         |
 
```

Comment:
Can you also add the https://github.com/paketo-buildpacks/builder-ubi8-buildpackless-base